typings/resin-image-fs.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'resin-image-fs' {
+	import Promise = require('bluebird');
+
+	export function read(options: {}): Promise<NodeJS.ReadableStream>;
```

Comment:
Not sure what you mean? We're already importing bluebird the line just above this.

Reply:
Ohhh my, I might be getting blind, or cross posted this here instead of the other file.
Sorry for the noise, forget about this :kissing: (<= that's me whistling)